src/test/fuzz/banman.cpp

```diff
@@ -93,6 +95,22 @@ FUZZ_TARGET_INIT(banman, initialize_banman)
                     ban_man.Discourage(ConsumeNetAddr(fuzzed_data_provider));
                 });
         }
+        if (!force_read_and_write_to_err) {
+            ban_man.DumpBanlist();
+            SetMockTime(ConsumeTime(fuzzed_data_provider));
+            banmap_t banmap;
+            ban_man.GetBanned(banmap);
+            BanMan ban_man_read{banlist_file, nullptr, offset};
```

Comment:
The 3rd argument `offset` is only used by `BanMan::Ban()` which we never call on `ban_man_read`. So this can be 
```suggestion
            BanMan ban_man_read{banlist_file, nullptr, 0};
```
and then the `offset` variable is not necessary.

Reply:
Thanks, done